### PR TITLE
Ubuntu-19.10:Bugfix libleveldb1v5 not support eoan

### PR DESCRIPTION
**Pull request by tkng5e**
Libleveldb1v5 do not have eoan support, changed to use official package

Remove 2 machines in ubuntu-19.10 test case to save ram during testing

### CODE REVIEW

File: molecule/ubuntu-19.10/molecule.yml [hunk 1]
```diff
@@ -38,44 +38,6 @@ platforms:
       - ceph-osd
       - ceph-mds
       - ceph-rgw
-  - name: ${MOLECULE_INSTANCE_NAME}-2
```
Comment (hswong3i) [inline molecule/ubuntu-19.10/molecule.yml:41]: @tkng5e please `git rebase -i` with https://github.com/alvistack/ansible-role-ceph-common/commit/b62c1441097dbe2b3e72192143bd69b200496a98

Comment (hswong3i) [reply molecule/ubuntu-19.10/molecule.yml:41]: The extra instances should already being removed from `develop`

Comment (hswong3i) [reply molecule/ubuntu-19.10/molecule.yml:41]: @tkng5e Oh please also remember to reword the commit message and PR comment ;-)

Comment (tkng5e) [reply molecule/ubuntu-19.10/molecule.yml:41]: should be ok now, plz check

**Pull request merged**